sources/packages/backend/apps/db-migrations/src/sql/DisbursementSchedules/Add-has-estimated-awards-col.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE
+    sims.disbursement_schedules
+ADD
+    COLUMN has_estimated_awards BOOLEAN NOT NULL DEFAULT TRUE;

```

Comment:
We must not default to true as there is production data already and there is potential to have actual sudent disbursement data when this code is going to be deployed. 

Please do the following

1. Set the default to false initially
2. Write an update query to calculate `TRUE | FALSE` based on the data.
3. Remove the default.